boxoffice/static/js/models/util.js

```diff
@@ -27,3 +32,58 @@ export const fetch = function(config){
     dataType: 'json'
   });
 }
+
+export const post = function(config){
+  return $.ajax({
+    url: config.url,
+    type: 'POST',
+    dataType: 'json'
+  });
+}
+
+export const TableSearch = function(tableId){
```

Comment:
Can we evaluate `http://www.listjs.com/`? `TableSearch` still has some optimization work left, but if listjs works well, we can consider switching.